config/environments/production.rb

```diff
@@ -34,7 +34,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for nginx
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
I'm wondering if this might be a problem when we deploy to AWS.  We don't do it in the portal.  We'll see this morning.